tests/test_views.py

```diff
@@ -0,0 +1,7 @@
+from app import app
+
+
+def test_animes_route():
+    client = app.test_client()
+    response = client.get('/api/v1/animes/')
+    assert response.status_code == 200
```

Comment:
I created a DB externally using sql browser with 3 records, and I changed the app.config["SQLALCHEMY_DATABASE_URI"]  in app.py to the fake DB, then the response is correctly displayed. I wanted to create new app in the test_views script for testing, but in views.py there is an import from app.py. Is it sufficient for the current task?

Reply:
... later to last comment above, last commit (# 08c09c3) was pushed to solve the issue in views.py by handling an import error. and now the testing goes flawless.

Reply:
> I created a DB externally using sql browser with 3 records, and I changed the app.config["SQLALCHEMY_DATABASE_URI"] in app.py to the fake DB, then the response is correctly displayed. I wanted to create new app in the test_views script for testing, but in views.py there is an import from app.py. Is it sufficient for the current task?

Running tests should not need manual steps since that's your first project I will let you decide. Honestly, I would prefer us to test this endpoint using factory boy with fake data. Once done, you will have an example to test other endpoints and the process will go smoothly.

Reply:
It can feel a bit overwhelming right now because you are learning new technologies but once this is acquired, you will go super fast on the rest of the work for this project.

Reply:
No worries I still can do it.

I will implement factory boy with fake data and push the changes.